pinning/pinner/dspinner/pin_test.go

```diff
@@ -890,11 +889,9 @@ func (d *batchWrap) Batch(_ context.Context) (ds.Batch, error) {
 }
 
 func makeStore() (ds.Datastore, ipld.DAGService) {
-	ldstore, err := lds.NewDatastore("", nil)
-	if err != nil {
-		panic(err)
-	}
-	dstore := &batchWrap{ldstore}
+	md := ds.NewMapDatastore()
```

Comment:
This have terrible Query performance and is totally unusable with the dht server code.

Reply:
I don't understand how this is relevant, given these are tests?

Reply:
I was under the impression this was a Kubo PR. It's fine here.